pkg/util/iptables/iptables.go

```diff
@@ -317,6 +318,9 @@ func (runner *runner) SaveInto(table Table, buffer *bytes.Buffer) error {
 	runner.mu.Lock()
 	defer runner.mu.Unlock()
 
+	trace := utiltrace.New("iptables save")
+	defer trace.LogIfLong(2 * time.Second)
```

Comment:
@wojtek-t curious how does this work ? is this running the iptables save command or that is just naming the trace ?

Reply:
It just adds a trace and ensures that if whole operation took more than 2s, it will be logged.